packages/native/src/components/LinearGradient.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
```

Comment:
did this generate by itself or did you create it by hand?

Reply:
@peter I made it.

Reply:
@peterpme please advise (should I keep it?)

Reply:
I think so!